### DISCUSSION TITLE
Set filemode for codestart extension project resources?

### DISCUSSION THREAD

**Discussion by joes** [quarkusio/quarkus#25946] [Q&A] [answered]
I am trying to include some helper bash-scripts in my codestart extension by adding them to `runtime/src/main/codestarts/quarkus/my-org-camel-integration-codestart/base/bin/init.tpl.qute.sh`.

I would like the generated file to be executable (`chmod +x`). 

Is this possible? Is there some `output-strategy` for `codestart.yml` that could be used to achieve this?

[a comment by an automated account is not shown]

**Comment by ia3andy** [comment, marked as answer]
https://github.com/quarkusio/quarkus/blob/main/independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/maven-wrapper/codestart.yml#L4
